M2/Macaulay2/packages/ConwayPolynomials.m2

```diff
@@ -10,24 +10,28 @@ newPackage(
     	Headline => "a database of Conway polynomials"
     	)
 -- the data comes libflint
-export "conwayPolynomial"
+export {
+    "conwayPolynomial",
+    -- options to set the variable to use
+    "Variable"
+    }
 rawConwayPolynomial := value Core#"private dictionary"#"rawConwayPolynomial"
 getCP := (p,n) -> rawConwayPolynomial (p,n,false)
-Ap := memoize(p -> (ZZ/p)(monoid [getSymbol "a"]))
+Ap := memoize((p, a) -> (ZZ/p)(monoid [a]))
 fix := (p,n,co,a) -> sum(#co, i -> co#i * a^i)
-conwayPolynomial = method()
-conwayPolynomial(ZZ,ZZ) := (p,n) -> (
+conwayPolynomial = method(Options=>{Variable=>getSymbol "a"})
```

Comment:
I think that if you run  ' installPackage "ConwayPolynomials" ', this change will lead to a problem -- the error message will be something like " can't determine package of symbol a ".  The solution will be to let the default value be the string "a", and run getSymbol later.

Reply:
I rewrite this part by following the code in `galois.m2`